<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/ResImages/MDriect.png</normaloff>:/new/prefix1/ResImages/MDriect.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/ResImages/MAbout.png</normaloff>:/new/prefix1/ResImages/MAbout.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent; /* 设置背景为透明 */
    border: none; /* 去除边框 */
	color: white; 
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tishi">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
    </property>
    <property name="text">
     <string>在此处输入歌曲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: transparent; /* 设置背景为透明 */
	color: white; 
    border: none; /* 去除边框 */
    selection-background-color: #0000FF; /* 可选：设置选中文本的背景颜色 */
    selection-color: #FFFFFF; /* 可选：设置选中文本的颜色 */
}</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QgroupBox {
    background-color: transparent; /* 设置背景为透明 */
    border: none; /* 去除边框 */
}</string>
    </property>
    <property name="title">
     <string>在线音乐列表</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>289</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="lcdNumber_time">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_program">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px; /* 滑块槽的高度 */
    background: #e2e3e4;
    margin: 2px 0;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop: 0 rgba(32, 47, 130, 255), stop: 0.5 rgba(32, 47, 130, 255),
    stop: 1 rgba(32, 47, 130, 255));
    border: 1px solid #5c5c5c;
    width: 18px; /* 滑块的宽度 */
    margin: -2px 0; /* 滑块的偏移 */
    border-radius: 9px;
}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_file">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/MOpenSong.png</normaloff>:/new/prefix1/ResImages/MOpenSong.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_last">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/Preious.png</normaloff>:/new/prefix1/ResImages/Preious.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pause">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/MPlay.png</normaloff>:/new/prefix1/ResImages/MPlay.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/MNext.png</normaloff>:/new/prefix1/ResImages/MNext.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_closesound">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/MSound.png</normaloff>:/new/prefix1/ResImages/MSound.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_end">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/ResImages/MStops.png</normaloff>:/new/prefix1/ResImages/MStops.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_voice">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent; /* 按钮默认透明 */
    border: none; /* 去除边框 */
    outline: none; /* 去除轮廓线 */
}

QPushButton:hover {
    background-color: red; /* 鼠标悬停时显示红色 */
    border: none; /* 悬停时也无边框 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/ResImages/MQuit.png</normaloff>:/new/prefix1/ResImages/MQuit.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_top">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton_about</zorder>
   <zorder>lineEdit_input</zorder>
   <zorder>pushButton_tishi</zorder>
   <zorder>pushButton_search</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>pushButton_close</zorder>
   <zorder>label_top</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
